extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/KinesisRecordSupplier.java

```diff
@@ -663,34 +660,35 @@ public String getEarliestSequenceNumber(StreamPartition<String> partition)
     return getSequenceNumber(partition, ShardIteratorType.TRIM_HORIZON);
   }
 
+  /**
+   * Use the API listShards which is the recommended way instead of describeStream
+   * listShards can return 1000 shards per call and has a limit of 100TPS
```

Comment:
https://docs.aws.amazon.com/kinesis/latest/APIReference/API_ListShards.html - says 1000TPS per stream. typo?

Reply:
https://docs.aws.amazon.com/streams/latest/dev/service-sizes-and-limits.html - says 100TPS